csharp_dotnetcore/1.Console-EchoBot/README.md

```diff
@@ -0,0 +1,35 @@
+This sample shows a simple echo bot and demonstrates the bot working as a console app using the console adapter.
```

Comment:
The ConsoleAdapter is no longer a part of the official library, so we should have a note indicating this is an example of an adapter that developers can use as a reference for implementing their own. @cleemullins or @vishwacsena, please chime in if there are other things you'd like to be indicated here.

Reply:
Put "sample" console adapter.  If we want to broaden the visibility, i'll leave it to @vishwacsena .